lib/providers/notifiers/auth_notifier.dart

```diff
@@ -40,10 +42,14 @@ class AuthNotifier extends ChangeNotifier {
       status = AuthStatus.locked;
     }
 
-    if (_passcode == null) {
+    if (_passcode == null && _passcodeActive) {
       status = AuthStatus.emptyPasscode;
     }
 
+    if(status == AuthStatus.emptyPasscode && !_passcodeActive) {
```

Comment:
Hmm.... what does this logic do? Empty pass code, but pass code not active leads to ->> locked?

Should this be || ?

Reply:
There is some trickery in bridging the previous flow of the app with the new stuff I've built. I plan to refactor this whole AuthStatus and AuthNotifier thing, but that would add even more time to the task. 

This specific "hack" makes sure that the passcode screen won't show up if passcode is disabled. It's not nice, but it would require quite a few changes in main.dart otherwise.